modules/widgets/contact-info.php

```diff
@@ -123,6 +124,10 @@ function widget( $args, $instance ) {
 				}
 			}
 
+			if ( '' != $instance['email'] ) {
```

Comment:
Let's check if `is_email` before linking-- if it's not a valid email, show it as plain text, but not clickable